src/AbstractRightsDropdown.php

```diff
@@ -75,13 +75,17 @@ protected static function isTypeEnabled(string $type): bool
      *
      * @return string
      */
-    public static function show(string $name, array $values): string
+    public static function show(string $name, array $values, array $params = []): string
     {
         // Flatten values
         $dropdown_values = [];
         foreach ($values as $fkey => $ids) {
-            foreach ($ids as $id) {
-                $dropdown_values[] = $fkey . "-" . $id;
+            if (is_array($ids) || is_string($fkey)) {
+                foreach ($ids as $id) {
+                    $dropdown_values[] = $fkey . "-" . $id;
+                }
+            } else {
+                $dropdown_values[] = $ids;
```

Comment:
What values are you sending here that need these changes ?

I am afraid supporting multiple format will complicate things and make this code hard to maintain.
Maybe we could call the function with the correctly formatted values instead ?